apps/dashboard/src/routes/dashboard/link/+page.svelte

```diff
@@ -0,0 +1,118 @@
+<script lang="ts">
+	import { modalStore } from '@skeletonlabs/skeleton';
+	import type { ModalSettings } from '@skeletonlabs/skeleton';
+	import { popup } from '@skeletonlabs/skeleton';
+	import CreateLinkModal from './CreateLinkModal.svelte';
+	import LinkDetailsModal from './LinkDetailsModal.svelte';
+	import { faEllipsisV, faAdd, faPencil, faLink, faMinus } from '@fortawesome/free-solid-svg-icons';
+	import Fa from 'svelte-fa';
+
+	export let data;
+	const { currentPrefix, linkData } = data;
+	const tableData = linkData ?? [];
+
+	const newLinkModal: ModalSettings = {
+		type: 'component',
+		component: {
+			ref: CreateLinkModal
+		},
+		response: async (suffix: string) => {
+			await fetch('link/', {
+				method: 'POST',
+				body: JSON.stringify({ prefix: currentPrefix, suffix: suffix }),
+				headers: { 'content-type': 'application/json' }
+			});
```

Comment:
The user has requested to create a new suffix link, if the response is successful, we want to update `tableData` with the new data (which is fetched during `export const load`). What is the standard approach for this?

Reply:
Hmm like resetting the page? Mimicking a refresh? Not sure if there is a svelte way to do that.

Reply:
Notes:
supabase real time as a way to refresh, svelte might have something for this.